xml/bp_troubleshooting_iscsi.xml

```diff
@@ -13,7 +13,10 @@
    <dm:release>SES 7</dm:release>
   </dm:docmanager>
  </info>
- <para>
-  TBD
- </para>
+ <sect1 xml:id="troubleshooting-iscsi-lock-errors">
+   <title>&igw; Lock Errors</title>
+ </sect1>
+ <sect1 xml:id="troubleshooting-iscsi-dashboard">
+   <title>Unable To Manage &igw; in the &dashboard;</title>
+ </sect1>
```

Comment:
These empty sections will soon run out of validation, right? Add TBD paras please